src/mainwindow.cpp

```diff
@@ -160,6 +161,13 @@ MainWindow::MainWindow(QWidget* parent)
     showTimelineAction->setShortcut(tr("Ctrl+T"));
     connect(showTimelineAction, &QAction::toggled, m_resultsPage, &ResultsPage::setTimelineVisible);
 
+    auto *prettifySymbolsAction = ui->viewMenu->addAction(tr("Prettify Symbols"));
```

Comment:
please also add a tooltip saying something like

"Replace fully qualified and expanded STL type names with their shorter and more commonly used equivalents. E.g. show <tt>std::string</tt> instead of <tt>std::basic_string<char, ...></tt>"